tests/integration/test_interface.py

```diff
@@ -68,53 +68,34 @@ def test_generate_dataset_and_col_noising(
     else:
         source = _generate_non_default_data_root(dataset.name, tmpdir, sample_data_path, data)
 
-    # Update unit number cell probabilities because there are so few of them
-    # that nothing gets noised with default levels
-    config = {
-        dataset.name: {
-            Keys.COLUMN_NOISE: {
-                column.name: {
-                    noise_type.name: {
-                        Keys.CELL_PROBABILITY: 0.25,
-                    }
-                    for noise_type in COLUMNS.unit_number.noise_types
-                }
-                for column in dataset.columns
-                if "unit_number" in column.name
-            },
-        },
-    }
+    # Increase cell probability
+    cell_probability = 0.25
+    custom_config = {dataset.name: {Keys.COLUMN_NOISE: {}}}  # initialize
+    for col in [c for c in dataset.columns if c.noise_types]:
+        custom_config[dataset.name][Keys.COLUMN_NOISE][col.name] = {
+            noise_type.name: {
+                Keys.CELL_PROBABILITY: cell_probability,
+            }
+            for noise_type in col.noise_types
+        }
 
     # Update SSA dataset to noise 'ssn' but NOT noise 'ssa_event_type' since that
     # will be used as an identifier along with simulant_id
     # TODO: Noise ssa_event_type when record IDs are implemented (MIC-4039)
     if dataset.name == DATASETS.ssa.name:
-        config[dataset.name][Keys.COLUMN_NOISE][COLUMNS.ssn.name] = {
-            noise_type.name: {
-                Keys.CELL_PROBABILITY: 0.01,
-            }
-            for noise_type in COLUMNS.ssn.noise_types
-        }
-        config[dataset.name][Keys.COLUMN_NOISE][COLUMNS.ssa_event_type.name] = {
+        custom_config[dataset.name][Keys.COLUMN_NOISE][COLUMNS.ssa_event_type.name] = {
             noise_type.name: {
                 Keys.CELL_PROBABILITY: 0,
             }
             for noise_type in COLUMNS.ssa_event_type.noise_types
         }
 
-    # FIXME: Do not set the row-level probability to zero
-    # Set row-level noise to zero since we're just testing columns here.
-    config[dataset.name][Keys.ROW_NOISE] = {
-        noise_type.name: {
-            Keys.ROW_PROBABILITY: 0,
-        }
-        for noise_type in dataset.row_noise_types
-    }
-
-    noised_data = noising_function(seed=0, source=source, year=None, config=config)
-    noised_data_same_seed = noising_function(seed=0, source=source, year=None, config=config)
+    noised_data = noising_function(seed=0, source=source, year=None, config=custom_config)
+    noised_data_same_seed = noising_function(
+        seed=0, source=source, year=None, config=custom_config
+    )
     noised_data_different_seed = noising_function(
```

Comment:
We do this check in the test column noise too.  Not sure if we could set it up to do for both modules but it is an idea.